rbd/snapshot_nautilus.go

```diff
@@ -66,3 +66,37 @@ func (image *Image) GetParentInfo(p_pool, p_name, p_snapname []byte) error {
 
 	return nil
 }
+
+// ssize_t rbd_list_children(rbd_image_t image, char *pools, size_t *pools_len,
```

Comment:
Since the original function had a proper doc comment I'd like not to lose it.
This implements rbd_list_children3

Reply:
Still relevant IMO